Some eMMC moduled that come with ROC-RK3399-PC don't cope well with HS400/HS400ES.
The modules that have issues are mentioned below DJNB4R and mine AJTD4R.

See:
https://lkml.org/lkml/2019/11/11/203
https://lkml.org/lkml/2019/11/15/444

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-roc-pc.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-roc-pc.dtsi
index 19f7732d7..bef364a09 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-roc-pc.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-roc-pc.dtsi
@@ -646,8 +646,7 @@
 
 &sdhci {
 	bus-width = <8>;
+	mmc-hs200-1_8v;
-	mmc-hs400-1_8v;
-	mmc-hs400-enhanced-strobe;
 	non-removable;
 	status = "okay";
 };
 
